Engine/Src/Ancona/Core2D/Systems/Collision/CollisionSystem.cpp

```diff
@@ -20,18 +21,15 @@ CollisionSystem::CollisionSystem(const std::string & name, SystemManager & manag
 void CollisionSystem::OnLoad()
 {
     CreateType(NONE_COLLISION_TYPE);
-    if (FileOperations::IsFile("project/collision-types.json"))
+    auto fileStream = FileOperations::GetInputFileStream("project/collision-types.json");
```

Comment:
We should instead fix the FileOperations::IsFile function on android to check for non apk storage